samples/security/spire/istio-spire-config.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       templates:
         spire: |
           spec:
-            containers:
+            initContainers:
```

Comment:
This will fix a subset of users while breaking another subset. No clue what percent, but its probably a non-trivial amount on each... we should probably at least document when you do or do not need to change this.

Or see if we can make the injector handle it ... @keithmattix